### ISSUE TITLE
u.Hostname undefined

### ISSUE THREAD

**Issue by Rudloff** [mattermost/platform#6434]
#### Summary
I get this error when trying to run the latest master:
```
# github.com/mattermost/platform/utils
utils/utils.go:77: u.Hostname undefined (type *url.URL has no field or method Hostname)
```

#### Steps to reproduce
`go run ./cmd/platform/*.go`

#### Expected behavior
Mattermost should start normally.

#### Observed behavior
Mattermost does not start.

#### Possible fixes
I think URL.Hostname has been added in Go 1.8 and I'm using 1.7.4 (the latest version available on Debian Jessie).

**Comment by jasonblais** [comment]
Thanks @Rudloff, we now recommend installing Go 1.8 for developer setups: https://docs.mattermost.com/developer/dev-setup.html

We are also aware of an issue caused by https://github.com/mattermost/platform/pull/6322 and are currently fixing it in the `master` branch. But the issue seems different from yours https://github.com/mattermost/platform/issues/6425

**Comment by lindy65** [comment]
Hi @Rudloff,

Circling back to see whether your issue is resolved with post above from jasonblais?

**Comment by Rudloff** [comment]
I still get the same issue on the latest master.
I understand if you only want to support Go >= 1.8, it is just easier for me if I can stick to packages available in Debian.

**Comment by cpanato** [comment]
per this doc: https://docs.mattermost.com/developer/dev-setup-troubleshooting.html Go 1.8 is supported.

**Comment by lindy65** [comment]
Hi @Rudloff,

Circling back on this issue.

If I understand correctly, you're still using Go 1.7.4 (the latest version available on Debian Jessie) so this issue won't be resolved for you as we support Go >= 1.8?

It seems we might have to wait for Debian Jessie to support Go 1.8 before your issue is resolved.

I'll close this issue off for now but let us know if there is anything we can help with in the meantime.
